controllers/configmap_controller.go

```diff
@@ -477,6 +478,28 @@ func (r *ConfigMapReconciler) EnsureServicesConfigMapExists() error {
 	return r.createServicesConfigMapOnBootup()
 }
 
+// UpdateWICDRoleBindingNamespace updates the WICD RoleBinding to point to the right ServiceAccount namespace
+func (r *ConfigMapReconciler) UpdateWICDRoleBindingNamespace(wicdResourceName string) error {
```

Comment:
`wicdResource` should not be an input parameter given it is something that will never change.